cmd/fossa/main.go

```diff
@@ -49,13 +47,23 @@ func main() {
 		cli.StringFlag{Name: "m, modules", Usage: "the modules to build and analyze"},
 		cli.BoolFlag{Name: "o, output", Usage: analyzeOutputUsage},
 		cli.BoolFlag{Name: "allow-unresolved", Usage: analyzeAllowResolvedUsage},
-		cli.BoolFlag{Name: "no-upload", Usage: analyzeNoUploadUsage},
 		cli.BoolFlag{Name: "b, build", Usage: "run a default build in module directories if they have not been pre-built"},
 		cli.BoolFlag{Name: "f, force", Usage: buildForceUsage},
 		cli.BoolFlag{Name: "debug", Usage: debugUsage},
 	}
 
 	app.Commands = []cli.Command{
+		{
+			Name:   "init",
+			Usage:  "Scans your environment for code module entry points and writes to config",
+			Action: initCmd,
+			Flags: []cli.Flag{
+				cli.BoolFlag{Name: "f, force", Usage: "rescan and overwrite modules in config even if they exist"},
```

Comment:
This doesn't seem like a real use case to me. I would remove this until someone requests it.